app/_layout.tsx

```diff
@@ -51,6 +52,42 @@ export default function HomeLayout() {
               },
             }}
           />
+          <Stack.Screen
+            name="notes/[noteId]"
+            options={{
+              presentation: 'modal',
+              title: '',
+              animation: 'slide_from_bottom',
+              headerTintColor: colors.text,
+              headerStyle: {
+                backgroundColor: colors.background,
+              },
+            }}
+          />
```

Comment:
Should `guests/[guestId]` and `notes/[noteId]` instead be regular screens instead of a modal?

Reply:
5abdcccbfb8a7819ea8d053f01284e42100be9eb